docs/examples/tensorflow/demo/nvutils/image_processing.py

```diff
@@ -73,7 +73,7 @@ def __init__(self,
 
         if training:
             if dali_cpu:
-                self.resize = dali.ops.Resize (device="cpu", resize_x=width, resize_y=heigth)
+                self.resize = dali.ops.Resize (device="cpu", resize_x=width, resize_y=height)
             else:
                 self.resize = dali.ops.RandomResizedCrop(
                     device=resize_device,
```

Comment:
Actually `resize_device` is not set anymore (after @jantonguirao 's changes).
Please set it as [it was set prior to the changes](https://github.com/NVIDIA/DALI/blob/b76919292f808a48d69aa9ac1d518bde51dbf274/docs/examples/tensorflow/demo/nvutils/image_processing.py#L64), or remove it

Reply:
Done